source/_integrations/telegram_webhooks.markdown

```diff
@@ -22,15 +24,17 @@ To integrate this into Home Assistant, add the following section to your `config
 telegram_bot:
   - platform: webhooks
     api_key: YOUR_API_KEY
-    parse_mode: html
+    trusted_networks:
```

Comment:
We try to keep all configuration samples minimal. Thus, no optional requirement in the default sample. This helps a user to get started quickly by copy-&-paste the sample without worrying about optional parameters which they most likely not need. If required, insert a full configuration sample later that covers special setups or alike.

Reply:
Changed accordingly 👍